Moli/Core/Screens/Profile/ProfileImageAndBannerView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Mantis
+
+enum ImageType {
+    case profile, banner
+}
+
+struct ProfileImageAndBannerView: View {
+    @EnvironmentObject private var myUser: MyUserDataModel
+    @StateObject var profileImagesDataModel = ProfileImagesDataModel()
+
+    var bannerImages: API.User.BannerThumbnailContainer?
+    var profileUserImage: API.User.UserMinified
+    var isMe: Bool
+    private let profileImageSizeYOffset: CGFloat = .userImageSizeMax / 2
+
+    private func onImageCropped (_ image: UIImage) {
+        Task {
+            await profileImagesDataModel.onImageCroppedSelected(image)
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    private func onRemoveImage () {
+        Task {
+            await profileImagesDataModel.removeImage()
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            HStack(spacing: 0) {
+                BannerImageView(bannerImages: bannerImages, isMe: isMe)
+            }
+            .frame(height: .bannerImageHeight)
+            .frame(maxWidth: .infinity)
+            .background(Color.gray50)
+            .onTapGesture {
+                if isMe {
+                    profileImagesDataModel.onImageTap(.banner)
+                }
+            }
+
+            VStack(spacing: 0) {
+                UserThumbnailView(userMinified: profileUserImage, size: .max)
+            }
+            .overlay(Circle().strokeBorder(Color.primaryDark, lineWidth: 3, antialiased: true))
+            .offset(x: 0, y: profileImageSizeYOffset)
+            .onTapGesture {
+                if isMe {
+                    profileImagesDataModel.onImageTap(.profile)
+                }
+            }
+        }
+        .padding(.bottom, profileImageSizeYOffset)
+        .confirmationDialog(
+            "Are you sure?",
+            isPresented: $profileImagesDataModel.shouldShowPhotoConfirmDialog
+        ) {
+            Button("Choose image from library") {profileImagesDataModel.chooseImageFromLibrary()}
+            .padding()
+
+            Button("Take a photo") { profileImagesDataModel.takeCameraImage() }
+            .padding()
+
+            Button("Remove current image", action: onRemoveImage).padding()
+        }
+        .fullScreenCover(isPresented: $profileImagesDataModel.isImagePickerDisplay) {
+            ImagePickerView(sourceType: profileImagesDataModel.sourceType, onImageSelected: profileImagesDataModel.onImageSelected)
```

Comment:
:bomb: Här vill vi faktiskt ignorera safe area för att få en "fullständig"-fullscreen beteende, då kan vi få till det riktigt snyggt, nedan lämnar jag skillnaden :P 

(Och .ignoresSafeArea(.all) är samma sak som .ignoresSafeArea() men personligen känner jag att det är bättre när man specifierar vad som man ignorerar, för ibland vill man bara ignorera keyboard:n tex)


```suggestion
            ImagePickerView(sourceType: profileImagesDataModel.sourceType, onImageSelected: profileImagesDataModel.onImageSelected)
            ..ignoresSafeArea(.all)
```
![IMG_6683](https://user-images.githubusercontent.com/10032344/150984696-ff5bc6d6-83fe-4a47-9ad8-f4bafb6bd16c.PNG)

![IMG_6684](https://user-images.githubusercontent.com/10032344/150984746-4a21e37a-7358-4062-a1f3-2c356d3b1b42.PNG)